### PR TITLE
Feat : create layouts and SEO

### PR DESCRIPTION
**Pull request by JeanneGrenet**
## Description : 
Create a generic layout and a specific layout for events and subpages events using generic layout
## How to manage SEO
With the [astro-seo](https://github.com/jonasmerlin/astro-seo) library, we can put a `<SEO />` tag when we use a layout with all the SEO props.

### CODE REVIEW

File: astro.config.mjs [hunk 1]
```diff
@@ -4,9 +4,14 @@ import tailwind from "@astrojs/tailwind";
 
 import mdx from "@astrojs/mdx";
 
+import sitemap from "@astrojs/sitemap";
```
Comment (houssembaltii) [inline astro.config.mjs:7]: can you run prettier command to format  this file

File: src/layouts/Layout.astro [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+const {title, description} = Astro.props
+---
+
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width">
+    <title>{title}</title>
+    <meta name="description" content={description || title}/>
```
Comment (houssembaltii) [inline src/layouts/Layout.astro:20]: same for this file

File: src/layouts/Layout.astro [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    <link rel="icon" href="/favicon.png" sizes="32x32" />
+    <link rel="sitemap" href="/sitemap-index.xml" />
+    <link rel="canonical" href={Astro.url} />
+    <meta name="generator" content={Astro.generator} />
+    <slot name="seo" />
```
Comment (houssembaltii) [inline src/layouts/Layout.astro:25]: can u add the pull request description , how are we gonna maange seo for each page using this slot .

**Review by houssembaltii — CHANGES_REQUESTED** [review]
* in the issue , we have rss feed that is missing here 
*  how to manage SEO for each page , i only see  `<slot name="seo"/>`  ,
* SEO :  open graph and twitter attributes are  missing 
* check prettier configuration , i think it's not formatting code on save .

File: src/layouts/Layout.astro (hunk 3 above)
Comment (JeanneGrenet) [reply src/layouts/Layout.astro:25]: Yes I added the astro-seo lib to manage it easily

**Pull request merged**